modules/network-observability-packet-translation-overview.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-packet-translation-overview_{context}"]
+= Endpoint translation (xlat)
+You can gain visibility into the endpoints serving traffic in a consolidated view using Network Observability and eBPF (extended Berkeley Packet Filter) to enrich Kubernetes service flows. Typically, when traffic flows through a Kubernetes Service, a kube-proxy or other load balancers abstracts the traffic flow information. For example, a request to `my-service` on `ClusterIP` is transparently routed to one of the available pods. However, often the information for both the service traffic and the virtual service endpoint is captured as two separate flows, which complicates troubleshooting. 
```

Comment:
Typically when traffic flows through a Service, EgressIP or Load balancer then the traffic is abstracted and routed to one of the availabel pods.

Reply:
However it does not provide granular info about which actual pod is serving that request

Reply:
We just service related info( service IP, port) and not the pod info